framework/sandstone.cpp

```diff
@@ -1079,21 +1085,24 @@ class QueueDeviceSchedule : public DeviceSchedule
 class RandomDeviceSchedule : public DeviceSchedule
 {
 public:
-    int get_next_cpu() override
-    {
-        // return random cpu
-        return random32() % num_cpus();
+    void device_reschedule() override {
+        // Select a random cpu index among the ones available
+        int next_cpu = random32() % num_cpus();
```

Comment:
Nitpick: `random()` because it's *slightly* faster. `unsigned(random())` for best performance.

Eventually we'll need to have a better ranged random, because this is not very uniform.

Reply:
Oh got it!